evaluation_plans/osps/legal/steps_test.go

```diff
@@ -8,6 +8,26 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+// FakeGraphqlRepo provides a minimal implementation of GraphqlRepoData
+// for testing OSPS-LE-03.02 license verification. It only includes
+// the LicenseInfo structure needed for license URL verification.
+type FakeGraphqlRepo struct {
+	Repository struct {
+		LicenseInfo struct {
+			Url string
+		}
+	}
+}
```

Comment:
@zohayb23 How does this compare to the alternative of using the full struct with only the necessary values populated?